api/app_sph_lms/models.py

```diff
@@ -1,379 +1,448 @@
+    tag = models.ForeignKey(Tag, on_delete=models.CASCADE)
+    created_at = models.DateTimeField(auto_now_add=True, null=True)
+    updated_at = models.DateTimeField(auto_now=True, null=True)
+
+    class Meta:
+        unique_together = ('course', 'tag')
+        verbose_name = "CourseTag"
+        verbose_name_plural = "CourseTag"
+        db_table = "app_sph_lms_course_tags"
+
+    def __str__(self):
+        return "Course: " + str(self.course) + " | " + "Tag: " + str(self.tag)
+
+
+class Lesson(models.Model):
+    course = models.ForeignKey(
+            Course,
+            on_delete=models.CASCADE,
+            related_name="lessons"
+        )
+    title = models.CharField(
+            max_length=255,
+            validators=[MinLengthValidator(3)]
+        )
+    link = models.URLField(max_length=600)
+    description = models.TextField(
+            max_length=65000,
+            null=True,
+            validators=[MinLengthValidator(5)]
+        )
+    is_active = models.BooleanField(default=True)
+    order = models.IntegerField(default=0)
+    created_at = models.DateTimeField(auto_now_add=True, null=True)
+    updated_at = models.DateTimeField(auto_now=True, null=True)
+
+    def __str__(self):
+        return self.title
+
+
+class CourseTrainee(models.Model):
+    trainee = models.ForeignKey(
+            Trainee, on_delete=models.CASCADE,
+            related_name="trainee_detail"
+        )
+    course = models.ForeignKey(
+            Course,
+            on_delete=models.CASCADE,
+            related_name="course_detail"
+        )
+    is_active = models.BooleanField(default=1)
+    created_at = models.DateTimeField(auto_now_add=True)
+    updated_at = models.DateTimeField(auto_now=True)
+
+    class Meta:
+        unique_together = ('trainee', 'course')
+        verbose_name = "CourseTrainee"
+        verbose_name_plural = "CourseTrainees"
+        db_table = "app_sph_lms_course_trainees"
+
+    def __str__(self):
+        return (
+            "Trainee: "
+            + str(self.trainee)
+            + " | "
+            + "Course: "
+            + str(self.course)
+        )
+
+
+class LearningPath(models.Model):
+    name = models.CharField(
+        max_length=255,
+        unique=True,
+        validators=[MinLengthValidator(3)]
+    )
+    author = models.ForeignKey(User, on_delete=models.CASCADE)
+    description = models.TextField(
+            max_length=65000,
+            null=True,
+            validators=[MinLengthValidator(5)]
+        )
+    image = models.CharField(max_length=255, null=True)
+    is_active = models.BooleanField(default=True)
+    created_at = models.DateTimeField(auto_now_add=True)
+    updated_at = models.DateTimeField(auto_now=True, null=True)
+
+    class Meta:
+        verbose_name = "LearningPath"
+        verbose_name_plural = "LearningPaths"
+        db_table = "app_sph_lms_learning_paths"
+
+    def __str__(self):
+        return self.name
+

```

Comment:
```suggestion
class LearningPath(models.Model):
    name = models.CharField(
        max_length=255,
        unique=True,
        validators=[MinLengthValidator(3)]
    )
    author = models.ForeignKey(User, on_delete=models.CASCADE)
    description = models.TextField(
            max_length=65000,
            null=True,
            validators=[MinLengthValidator(5)]
        )
    image = models.CharField(max_length=255, null=True)
    is_active = models.BooleanField(default=True)
    courses = models.ManyToManyField(Course, related_name="learning_path", through='LearningPathCourse')
    category = models.ManyToManyField(Category, related_name="category")
    created_at = models.DateTimeField(auto_now_add=True)
    updated_at = models.DateTimeField(auto_now=True, null=True)

    class Meta:
        verbose_name = "LearningPath"
        verbose_name_plural = "LearningPaths"
        db_table = "app_sph_lms_learning_paths"

    def __str__(self):
        return self.name


```

Please add many to many relations with  category and courses in the model, and also properly get the values from the relations in your serializer